### PR TITLE
MAINT-27935: Images are displayed in Documents widget

### PR DESCRIPTION
**Pull request by AzmiTouil**
In Documents widget in the snapshot page, images (png, jpeg, jpg..) appear in the "My work" tab. it shouldn't be the case, only files having these extensions **doc, docx, xls, xlsx, ppt, pptx, odt, rtf, txt and pdf** should appear according the the spec done in DW FT

### CODE REVIEW

File: core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java [hunk 1]
```diff
@@ -117,11 +117,17 @@ private ElasticSearchFilter getFileTypesFilter() {
...
-    fileTypesFilter.append(",{\n \"term\" : { \"fileType\" : \"image/gif\" }\n }");
+    if (!myWork) {
+      fileTypesFilter.append(",{\n \"term\" : { \"fileType\" : \"image/jpeg\" }\n }");
+      fileTypesFilter.append(",{\n \"term\" : { \"fileType\" : \"image/png\" }\n }");
+      fileTypesFilter.append(",{\n \"term\" : { \"fileType\" : \"image/gif\" }\n }");
+    }
     return new ElasticSearchFilter(ElasticSearchFilterType.FILTER_CUSTOM, "fileType", fileTypesFilter.toString());
   }
+
+  private ElasticSearchFilter getFileTypesFilter() {
```
Comment (ahamdi) [inline core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java:128]: You can remove this function as it is private to this class and no more used .

Comment (AzmiTouil) [reply core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java:128]: it's still used in https://github.com/exodev/ecms/blob/ab15f871a3a2df5a2966964335253b39fb33c5bf/core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java#L97

File: core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public Response searchRecentDocuments(@ApiParam(value = "Query string", required
     if (myWork) {
       recentFilters.add(filterMyWorkingDocuments());
     }
-    recentFilters.add(getFileTypesFilter());
+    recentFilters.add(getFileTypesFilter(myWork));
     Identity currentIdentity = ConversationState.getCurrent().getIdentity();
     UserACL userACL = PortalContainer.getInstance().getComponentInstanceOfType(UserACL.class);
     if (!currentIdentity.isMemberOf(userACL.getAdminGroups(), "*")) {
```
Comment (ahamdi) [inline core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java:80]: Could you change this to : 
```suggestion
    if ( !userACL.isSuperUser() && !userACL.isUserInGroup(userACL.getAdminGroups()))) {
```

**Review by azayati — APPROVED** [review]
LGTM after upading Ali's last suggested change

File: core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java (hunk 2 above)
Comment (AzmiTouil) [reply core/search/src/main/java/org/exoplatform/services/wcm/search/FileSearchRestService.java]: Updated !

**Review by azayati — APPROVED** [review]
LGTM

**Review by ahamdi — APPROVED** [review]

**Pull request merged**